src/views/home.hbs

```diff
@@ -1,12 +1,10 @@
-
 <h1>hello </h1>
 <ul>
     {{#each data}}
-    <li>{{this.name}}
-<img src="{{this.img}}"/>
-    </li>
-
+    <a href="/flower/{{this.id}}" <li>{{this.name}}
```

Comment:
Another suggestion to get this done is creating a helpers folder and create a helper function that takes id and returns the endpoint like: 
```js 
const getFlower= (id) =>{
 return '/flower/${id}'
 }
```
It would be better to use this way cos you may have more than one helper so anyone can find all the functions at the same place

Reply:
we refactor it as you mention @shahenazmonia